app/assets/javascripts/controllers/ansible_credentials/ansible_credentials_form_controller.js

```diff
@@ -58,13 +58,13 @@ ManageIQ.angular.app.controller('ansibleCredentialsFormController', ['$window',
 
   vm.saveClicked = function(angularForm) {
     API.put('/api/authentications/' + credentialId, vm.credentialModel)
-       .then(getBack(sprintf(__("Modification of Credential \"%s\" has been successfully queued."), vm.credentialModel.name), false))
+       .then(getBack.bind(vm, sprintf(__("Modification of Credential \"%s\" has been successfully queued."), vm.credentialModel.name), false))
```

Comment:
@AparnaKarve There is a difference between giving `.then` a function and a result of a function. If you give it a function it runs it after it gets response from API call. But if you give it a result of a function as it was before this fix it runs the function to get the result before it gets response from API. So it generates error because there is no response but at the same time it runs success function `getBack` to get its result into `.then`. So perils of asynchronous programming :) I hope I got @himdel explanation right :D